src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall() {
+        Random random = new Random();
+        ColorSupplier colorSupplier = new ColorSupplier();
+        return new Ball(colorSupplier.getRandomColor(), random.nextInt(100)).toString();

```

Comment:
make 100 a class constant, use CONSTANT CASE
Also, let's change it to 101 (passed value is excluded) - to make 100 a maximum possible number